app/pages/organization/stats/components/BarChart/components/Parameters.jsx

```diff
@@ -0,0 +1,218 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Translate from 'react-translate-component';
+import styled from 'styled-components';
+
+import filterTimeSeries from '../helpers/filterTimeSeries';
+import formatLabel from '../helpers/formatLabel';
+import Select from './Select';
+
+const StyledParametersContainer = styled.div`
+  background-color: #EFF2F5;
+  border-top: 1px solid #A6A7A9;
+  padding: 1.4em;
+`;
+
+const StyledTitle = styled.h4`
+  color: #5C5C5C;
+  font-weight: bold;
+  letter-spacing: 1.5px;
+  line-height: 17px;
+  margin-bottom: 1.2em;
+  text-transform: uppercase;
+`;
+
+const StyledInputsContainer = styled.div`
+  align-items: center;
+  display: flex;
+  height: 2.5em;
+`;
+
+const StyledText = styled.span`
+  color: #000000;
+  white-space: nowrap;
+`;
+
+const StyledResetContainer = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: row-reverse;
+  height: 2.5em;
+  justify-content: space-between;
+  margin-top: 1em;
+`;
+
+const StyledResetButton = styled.button.attrs({
+  activeClassName: 'active'
```

Comment:
Please don't use CSS classes to indicate state. Prefer ARIA roles and state instead eg. `role='radio' aria-checked='true'` or `role='checkbox' aria-checked='true'` then select on the ARIA state in your styles.

Reply:
I wasn't even using activeClassName 🤦‍♂. In a subsequent PR I'll add a gradient to the button, which will be different when active, and will implement accessibly, but for this PR I removed.